howso/engine/trainee.py

```diff
@@ -1208,20 +1137,12 @@ def predict(
             DataFrame consisting of the discriminative predicted results.
         """
         if action_features is None:
-            if self.default_action_features is None:
-                raise HowsoError(
-                    "No action features specified and no default action features are present. "
-                    "Please specify the action feature or add default action features "
-                    "to the Trainee."
-                )
-            else:
-                action_features = self.default_action_features
+            raise HowsoError(
+                "No action features specified. Please specify the action feature."
```

Comment:
Should update typehints/docstring to make action_features no longer optional.
```suggestion
                "No action features were specified. Please provide a value for `action_features`."
```